_includes/azerothcore/sidebar.html

```diff
@@ -76,6 +76,7 @@
             <li><a href="/wiki/remote-access">Remote Access</a></li>
             <li><a href="/wiki/waypoints-information">Information about waypoints</a></li>
             <li><a href="/wiki/how-to-obtain-free-records">How to obtain free records</a></li>
+	    <li><a href="/wiki/dbc-index">DBC index</a></li>
```

Comment:
no. Add to https://www.azerothcore.org/wiki/documentation-index

Reply:
Changed.